axon-framework/modules.md

```diff
@@ -4,13 +4,15 @@ Axon Framework consists of a number of modules that provide specific capabilitie
 
 There are currently two ways of obtaining the module binaries: either download the binaries from our website or preferably configure a repository for your build system \([Maven](http://maven.apache.org/), [Gradle](https://gradle.org/)\).
 
+To not be bothered with version compatibility issues between [framework](#main-modules) and the [extensions](#extension-modules), it is recommended to use the [BOM](#axon-bill-of-materials).
+
 Axon modules are available on [Maven Central](https://search.maven.org/search?q=axonframework).
 
 ## Main modules
 
 Axon 'Main Modules' are the modules that have been thoroughly tested and are robust enough to use in demanding production environments. The maven `groupId` of all these modules is `org.axonframework`. Visit [Maven Central Repository](https://search.maven.org/search?q=g:org.axonframework) to copy coordinates for the version you need.
 
-> **Note**
+> **Quick starting an Axon Application**
```

Comment:
```suggestion
> **Quick starting an Axon Application**
```
I think it would make more sense to say "Quickly Start and Axon Application" or "QuickStart an Axon Application". I think QuickStart is written like so, but please correct me if it's written like "Quick Start." I've seen both, but not entirely sure which one is best used here, as QuickStart refers to the loading method used by several applications, in order to speed up the loading time of an application... so, technically this can be the correct term for us I think, but I'm not 100% sure.

Reply:
It seems "the interwebs" isn't overly conclusive on the matter either... I see "Quickstart", "Quick Start", "QuickStart" and "Quick-Start" being used in equal amounts.
I'll go with your first suggestion here @SaraTorrey, as the at more closely aligns the wording we use in other parts of the ref-guide.